src/LibExtism.php

```diff
@@ -38,41 +39,42 @@ function findSo(string $name): \FFI {
         throw new \RuntimeException('Failed to find shared object. Searched locations: ' . implode(', ', $searchedPaths));
     }
 
-    function soname() {
+    public function soname()
+    {
         $platform = php_uname("s");
         switch ($platform) {
             case "Darwin":
                 return "libextism.dylib";
-            case "Linux": 
+            case "Linux":
                 return "libextism.so";
             case "Windows NT":
                 return "extism.dll";
             default:
-                throw new \Exception("Extism: unsupported platform ".$platform);
+                throw new \Exception("Extism: unsupported platform " . $platform);
         }
     }
 
-    function extism_current_plugin_memory(FFI\CData $plugin): \FFI\CData
+    public function extism_current_plugin_memory(FFI\CData $plugin): \FFI\CData
     {
         return $this->ffi->extism_current_plugin_memory($plugin);
     }
 
-    function extism_current_plugin_memory_free(FFI\CData $plugin, FFI\CData $ptr): void
+    public function extism_current_plugin_memory_free(FFI\CData $plugin, FFI\CData $ptr): void
```

Comment:
does this expose the methods to the external users? This is an internal API.

Reply:
They were already public if you not defined them explicitly protected or private.

From php.net
```
Class methods may be defined as public, private, or protected. 
Methods declared without any explicit visibility keyword are defined as public.
```

Reply:
Due to this function called in CurrentPlugin, we can't make it private.

Reply:
okay, just making sure we expose as little API to the public as possible. Maybe the class should be less accessible? I don't know if there is an equivalent of package only. But I'll leave @mhmd-azeez to pick out any changes that are needed. Thanks so much for this cleanup!

Reply:
Yeah, it seems like there is no package level access modifier in php. As a middle ground, I moved `LibExtism.php` to `\Extism\Internal` namespace, which is a pattern used by some php libraries like Symfony